ssg/build_yaml.py

```diff
@@ -634,9 +686,41 @@ def to_xml_element(self, env_yaml=None):
         ]
         groups_in_group = reorder_according_to_ordering(groups_in_group, priority_order)
         for group_id in groups_in_group:
+            if group_id in groups_to_not_include:
+                continue
             _group = self.groups[group_id]
             if _group is not None:
-                group.append(_group.to_xml_element(env_yaml))
+                group.append(_group.to_xml_element(env_yaml, components_to_not_include))
+
+    def to_xml_element(self, env_yaml=None,  components_to_not_include=None):
+        if components_to_not_include is None:
+            components_to_not_include = {}
+
+        rules_to_not_include = components_to_not_include.get("rules", set())
+        groups_to_not_include = components_to_not_include.get("groups", set())
+
+        if self.id_ in groups_to_not_include:
+            return None
+
+        group = self._create_group_xml_skeleton()
+
+        # This is where references should be put if there are any
+        # This is where rationale should be put if there are any
```

Comment:
???

Reply:
I left these comments in the method because they were present in the old version [`to_xml_element`](https://github.com/ComplianceAsCode/content/blob/9c496b1662d0690af13bbb32856b979e6b6f9a65/ssg/build_yaml.py#L561).